File: backend/src/MyDatingProfileHelper.Backend.Domain/Models/News.cs
csharp
using System;

namespace MyDatingProfileHelper.Backend.Domain.Models
{
    /// <summary>
    /// Представляет новость или информационное сообщение для пользователей.
    /// </summary>
    public class News
    {
        /// <summary>
        /// Уникальный идентификатор новости (первичный ключ).
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        /// Заголовок новости.
        /// </summary>
        public string Title { get; set; }

        /// <summary>
        /// Содержимое новости.
        /// </summary>
        public string Content { get; set; }

        /// <summary>
        /// Дата и время публикации новости.
        /// </summary>
        public DateTime PublishDate { get; set; }
\n+        /// <summary>\n+        /// Код языка новости (например, 'ru', 'en').\n+        /// </summary>\n+        public string LanguageCode { get; set; }\n    }\n}\n
    }
}

File: app/src/main/java/com/jounik_projects/mydatingprofilehelper/data/model/News.kt
package com.jounik_projects.mydatingprofilehelper.data.model

/**
 * Data class, представляющий новость.
 */
data class News(
    /**
     * Уникальный идентификатор новости.
     */
    val id: Int,
    /**
     * Заголовок новости.
     */
    val title: String,
    /**
     * Содержание новости.
     */
    val content: String,
    /**
     * Дата публикации новости в формате строки.
     */
    val publishDate: String // Для простоты используем String, можно использовать Date или LocalDateTime
)
